### PR TITLE
Nar 640 taxonomy replace

### PR DESCRIPTION
**Pull request by lina1101**

### CODE REVIEW

**Review by halge — APPROVED** [review]

File: sokannonser/rest/endpoint/platsannonser.py [hunk 1]
```diff
@@ -72,6 +72,19 @@ def get(self, **kwargs):
                         if max_score > 0 else 0.0})
                 for hit in result.get('hits', [])]
 
+        for hit in hits:
```
[the remark this answers is not in the document]
Comment (oskardrenske-ith) [reply sokannonser/rest/endpoint/platsannonser.py:75]: I added checks

**Review by oskardrenske-ith — APPROVED** [review]

File: sokannonser/repository/elastic_connection_with_retries.py [hunk 2]
```diff
@@ -12,7 +12,7 @@
 
 log = logging.getLogger(__name__)
 
-SLEEP_LENGTH = 0.2
+SLEEP_LENGTH = 2
```
Comment (halge) [inline sokannonser/repository/elastic_connection_with_retries.py:15]: add en envvar.

File: sokannonser/repository/elastic_connection_with_retries.py [hunk 3]
```diff
@@ -12,7 +12,7 @@
...
 ELASTIC_RETRIES = 2
```
Comment (halge) [inline sokannonser/repository/elastic_connection_with_retries.py:16]: add en envvar.

File: sokannonser/rest/endpoint/platsannonser.py [hunk 4]
```diff
@@ -72,6 +72,30 @@ def get(self, **kwargs):
...
+            occupations = hit.get('occupation', None)
```
Comment (halge) [inline sokannonser/rest/endpoint/platsannonser.py:76]: is there unit tests for None and not-none? if not add #Todo

**Review by halge — CHANGES_REQUESTED** [review]

**Review by halge — APPROVED** [review]

**Review by halge — APPROVED** [review]

**Pull request merged**